### ISSUE TITLE
The loss will be NAN while using the CUB scripts.

### ISSUE THREAD

**Issue by yhydhx** [hbdat/cvpr20_IMCL#4]
I downloaded the CUB data and models. 
In the code, it uses resnet_v1, however, you provided resnet_v2 in the README file. That's not the big problem, when I run the code and I figure out that the loss always turn to nan. When I run it step by step, the "validate_Prediction_val" was nan in the line 510 . 
Could you help me for this. Many thanks.

**Comment by hbdat** [comment]
Hi @yhydhx ,

Thanks for reaching out.
Did you pretrained the network with: `python ./CUB_experiments/e2e_baseline_logistic_CUB.py` before running the `python ./CUB_experiments/e2e_interactive_learning_CUB.py`?

Best,
Dat

**Comment by yhydhx** [comment]
Hi @hbdat ,
Thank you for your reply. Actually I tried both and both gave me nan. 
Also, the 36th line in the script "./CUB_experiments/e2e_baseline_logistic_CUB.py" looks useless. 
> "df_image = pd.read_csv('./data/2017_11/train/images.csv')"

Best,
Harold.

**Comment by hbdat** [comment]
Hi @yhydhx 

I am not sure while this is the case.
Have you tried to reduce the learning rate?

Also, thanks for pointing out the redundant code.

Best,
Dat

**Comment by yhydhx** [comment]
Hi @hbdat ,
That's a good advice, we will try it soon. 
Currently we are testing COCO Dataset and we figure out some tiny problem. 

In the file "e2e_interactive_learning_MSCOCO.py", line 342, "saver_init.restore(sess,NFS_path+'results/cont_ss_OpenImage_log_GPU_6_1552076726d777008/model_ES.ckpt')". Is this model from the baseline of COCO? 
Also there is no "results" folder in the root directory. That might cause problem and the program cannot save the result. 
Appreciate your feedback.

Best,
Harold.

**Comment by hbdat** [comment]
Hi @yhydhx,

Yes, the `results/cont_ss_OpenImage_log_GPU_6_1552076726d777008/model_ES.ckpt` is the pretrained model on MSCOCO.
Also, thanks for letting me know missing folder.
I have added it to the repo.

Best,
Dat
